test/parallel/test-util-inspect.js

```diff
@@ -288,17 +288,21 @@ assert.equal(util.inspect(setter, true), '{ [b]: [Setter] }');
 assert.equal(util.inspect(getterAndSetter, true), '{ [c]: [Getter/Setter] }');
 
 // exceptions should print the error message, not '{}'
-assert.equal(util.inspect(new Error()), '[Error]');
-assert.equal(util.inspect(new Error('FAIL')), '[Error: FAIL]');
-assert.equal(util.inspect(new TypeError('FAIL')), '[TypeError: FAIL]');
-assert.equal(util.inspect(new SyntaxError('FAIL')), '[SyntaxError: FAIL]');
+var errors = [];
```

Comment:
Another vote for `const`.  A value is not its referent; knowing that a referent cannot be reassigned makes it easier to keep the program state in your head.  The current rule of thumb in core is (or should be): `const` if you can, `let` or `var` if you must.